docs/releases/release-8-3.md

```diff
@@ -0,0 +1,417 @@
+- Plus, links to our extensive [Guest UEFI Secure Boot guide](../guides/guest-UEFI-Secure-Boot.md) are provided in the user interface where needed.
+
+We also fixed an issue — reported by one of our testers — that caused all UEFI VMs to fail on some specific hardware, and we sent the patch to XenServer developers.
+
+### Certificate verification [XS]
+
+While hosts within a pool are supposed to communicate over a secure, dedicated management network, the risk of a Man-in-the-Middle attack is never zero. To fully protect pools, XenServer developers added the *Certificate Verification* feature. This ensures that "all TLS communication endpoints use certificates to validate the identity of their peers before transmitting confidential information." (Quoting XenServer documentation.)
+
+:::note
+The feature is enabled for new installations but not enforced on upgraded pools. We strongly advise enabling it after the upgrade.
+
+Enable it with: `xe pool-enable-tls-verification`.
+:::
+
+:::warning
+If you don't enable it after an upgrade, you may encounter situations where you try to add a new, freshly installed host to a pool, and it fails due to `POOL_JOINING_HOST_TLS_VERIFICATION_MISMATCH`. This happens because the new host needs to verify the identity of its peers, but the pool doesn't support it yet. To resolve this, enable TLS verification on the pool.
+:::
+
+More about this feature can be found [in XenServer's documentation](https://docs.xenserver.com/en-us/xenserver/8/hosts-pools/certificate-verification).
+
+### OpenSSH, security, ciphers, and user configuration
+
+Accepted ciphers, keys and algorithms used to be defined in `/etc/sshd_config` and `/etc/ssh_config`. Whenever we wanted to update them, we had to choose between potentially overwriting user modifications to these files, and not updating them (actually we had a middle ground where we attempted to patch just the relevant lines, but success was not guaranteed).
+
+Now it's simpler:
+- We define the accepted ciphers, keys, and algorithms directly at OpenSSH build time.
+- As long as users do not define settings that take precedence, we use our built-in defaults. To update them, we just publish a new build.
+- Users who choose to modify their SSH configuration will need to maintain these settings themselves, as any updates provided by our Security Team through new builds will be overridden by user-defined settings.
+
+### Restricted use of port 80 [XS]
+
+To enhance security, it is now possible to close TCP port 80 on the management interface. By default, port 80 remains open for specific clients that may still require it, but all internal connections now utilize HTTPS. Previously, VM migration occurred over port 80, but this has been updated.
+
+To close port 80, run the following command:
+```
+xe pool-param-set uuid=<pool-uuid> https-only=true
+```
```

Comment:
Should we add a mention that this won't do it for IPv6 for now? Or do we expect the fix to be ready by release time?

Reply:
Let's mention it in the known issues section.